mne/io/persyst/persyst.py

```diff
@@ -74,7 +74,10 @@ def __init__(self, fname, preload=False, verbose=None):
         fileinfo_dict = OrderedDict()
         channelmap_dict = OrderedDict()
         patient_dict = OrderedDict()
-        comments_dict = OrderedDict()
+        comments_dict = defaultdict(list)
```

Comment:
The use of `OrderedDict` is intentional to a) make it clear it needs to be ordered and b) ordering only became an official language feature in 3.7 so it's safer to use OrderedDict while we still support 3.6 (other implementations like PyPy are not required to be ordered until then)